SERVER/api/v1/models/adModel.js

```diff
@@ -40,6 +41,39 @@ class CarSaleAd{
         return queryResult;
 
     }
+
+    // Return all cars (sold & available)
+    async allCars(){
+        const queryText = 'SELECT * FROM ads'; 
+        const queryResult = await db.query(queryText) 
+        return queryResult;
+    }
+
+    // Return a specific car sale ad. 
+
+    async specificAd(id){
```

Comment:
Apply the same idea you used in the last PR to have `allCars` and `specificAd` as one function to avoid repetition.

Reply:
@Kasulejoseph, I'm done applying the feedback on this branch. Thanks.